db/011_create_participants_table.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS `participants` (
+    `id` bigint unsigned NOT NULL AUTO_INCREMENT,
+    `event_id` bigint unsigned NOT NULL,
+    `user_id` bigint unsigned NOT NULL,
+    `status` varchar(20) CHARACTER SET ascii COLLATE ascii_general_ci DEFAULT 'INTERESTED',
+    `role` varchar(20) CHARACTER SET ascii COLLATE ascii_general_ci DEFAULT 'VOLUNTEER',
+    `created_at` datetime DEFAULT NULL,
+    `updated_at` datetime DEFAULT NULL,
```

Comment:
imo we can add ` DEFAULT CURRENT_TIMESTAMP` for `created_at` and 
`DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP` for `updated_at`.

Reply:
we can, but gorm is handling this for us